lector/resources/raw/io.github.BasioMeusPuga.Lector.metainfo.xml

```diff
@@ -0,0 +1,116 @@
+        </screenshot>
+
+        <screenshot>
+            <caption>Table view</caption>
+            <image>https://camo.githubusercontent.com/66cb4f09a81d164af50180eaa024f8e116c83815/68747470733a2f2f692e696d6775722e636f6d2f6f39416e3741522e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Book reading view, in optional Solarized display profile</caption>
+            <image>https://camo.githubusercontent.com/1b60a0c27383715e59ccfd9d2585544d303e08d9/68747470733a2f2f692e696d6775722e636f6d2f495447363346632e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Distraction-free view</caption>
+            <image>https://camo.githubusercontent.com/3289abadb668ad3e53b8ba779b52768b14b25bcf/68747470733a2f2f692e696d6775722e636f6d2f67384c747570792e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Annotation support</caption>
+            <image>https://camo.githubusercontent.com/80e7423b488e59ed2c05e556b4918c25522edfd1/68747470733a2f2f692e696d6775722e636f6d2f674c4b323946342e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Comic reading view</caption>
+            <image>https://camo.githubusercontent.com/81aaa963c6e9ed19a1605e2f2b017ce8c83ca656/68747470733a2f2f692e696d6775722e636f6d2f7276765451434d2e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Bookmark support</caption>
+            <image>https://camo.githubusercontent.com/ebbaffedcbd4f743ba9d67e5eaab2108407832cd/68747470733a2f2f692e696d6775722e636f6d2f5937716f55386d2e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Customizable view profiles</caption>
+            <image>https://camo.githubusercontent.com/0cea5e37e5c40abd4988544ba32b187db3cbe783/68747470733a2f2f692e696d6775722e636f6d2f6177453271324b2e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Metadata editor</caption>
+            <image>https://camo.githubusercontent.com/15362fb47e79b2d50eef507133cee8de6c1baa5c/68747470733a2f2f692e696d6775722e636f6d2f304344704e4f382e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>In-app dictionary</caption>
+            <image>https://camo.githubusercontent.com/de3bfc5359db85da8ad94850d12fe563b5eefb87/68747470733a2f2f692e696d6775722e636f6d2f524637326d32682e706e67</image>
+        </screenshot>
+
+        <screenshot>
+            <caption>Settings window</caption>
+            <image>https://camo.githubusercontent.com/9ef376164f311cf518b4825684e7d5cc9d1f538c/68747470733a2f2f692e696d6775722e636f6d2f6c367a4a5861482e706e67</image>
+        </screenshot>
+    </screenshots>
+
+    <url type="homepage">https://github.com/BasioMeusPuga/Lector</url>
+    <url type="bugtracker">https://github.com/BasioMeusPuga/Lector/issues</url>
+
+    <provides>
+        <binary>lector</binary>
+    </provides>
+
+    <releases>
```

Comment:
This section is basically just a changelog for the application. I include some dummy text here so you can see what the format is like, but the AppStream documentation calls it a "suggested" bit of information, rather than mandatory; I don't see very many apps in GNOME Software that actually includes this on Fedora at least. If you couldn't find a convenient way to generate the XML here based on a single source changelog file, or else if you didn't feel like editing this manually with every Lector release, I could live without this block 😜